data/obs-packaging/README.rst

```diff
@@ -5,31 +5,33 @@ Intel® Clear Containers OBS specification files
 ###############################################
 
 
-The `Clear Containers`_ packages are available via `Open Build Service`_
+The `Intel® Clear Containers`_ packages are available via `Open Build Service`_
 to allow interested parties the opportunity to try out the technology even
 if they are not using `Clear Linux` directly.
 
 When the `cc-oci-runtime`_ or any of its components are updated, we release a
 new package or set of packages.
 
+This repo only contains the `OBS`_ sources to build `cc-oci-runtime`_ and remaining items 
```

Comment:
This repo only contains the `OBS`_ sources to build the `cc-oci-runtime`_ and the remaining items